src/lib/parser/index.ts

```diff
@@ -37,7 +38,7 @@ const replaceWord = (str: string): string => {
     return str.replace(regex, "").trim();
 };
 
-function parseData (data: FilesContent, context: Context): RepoIssue[] {
+function parseData (data: FilesContent, context: any): RepoIssue[] {
```

Comment:
`Context` has a precise type. It shouldn't be set to `any`

Reply:
I think it would be better if this function just took in an author name argument, instead of context.